src/org/team4909/bionic/utils/subsystems/BionicVelocitySubsystem.java

```diff
@@ -25,7 +25,7 @@ public BionicVelocitySubsystem(CANTalon speedController) {
 	}
 	
 	public BionicVelocitySubsystem(CANTalon speedController, boolean inverted) {
-		this(speedController);
+		this(speedController, 0, 0, 0, 0);
```

Comment:
these shouldn't default to zeros... put the four input vals between the CANTalon and the bool